server/test/user/user.e2e-spec.ts

```diff
@@ -0,0 +1,297 @@
+import {HttpModule, INestApplication} from '@nestjs/common';
+import {Test} from '@nestjs/testing';
+import request from 'supertest';
+import {UserModule} from '../../src/port/module/user.module';
+import {User} from '../../src/domain/user/user.entity';
+import {DatabaseModule} from '../../src/port/module/database.module';
+import {ConfigModule, ConfigService} from 'nestjs-config';
+import * as path from 'path';
+import {getConnection} from 'typeorm';
+import * as assert from 'assert';
+
+// todo: implement ExceptionHandler, apply custom Error
```

Comment:
It seems to be possible to perform the test with one user data.

Reply:
i minimized the default data, but needed 2 data. That's because i needed 2 state: user with 10 point, user with 0 point